entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/Entity.java

```diff
@@ -53,7 +54,7 @@
     groups = {EntityFieldsEuropeanaProxyValidationGroup.class})
 @EntityFieldsDataSourceProxyValidationInterface(
     groups = {EntityFieldsDataSourceProxyValidationGroup.class})
-public abstract class Entity {
+public abstract class Entity implements ValidationEntity {
```

Comment:
should not extend ValidationEntity, the new interface can be added to the list of implemented Interfaces

Reply:
the interface is a technical not a functional interface